components/list/__tests__/__snapshots__/pagination.test.js.snap

```diff
@@ -221,9 +220,7 @@ exports[`List.pagination should change page size work 1`] = `
             aria-activedescendant="rc_select_TEST_OR_SSR_list_0"
             aria-autocomplete="list"
             aria-controls="rc_select_TEST_OR_SSR_list"
-            aria-expanded="false"
             aria-haspopup="listbox"
-            aria-label="Page Size"
```

Comment:
Why remove these?

Reply:
I just ran `yarn test -u` to update the snapshot.

Reply:
I have removed these updated snapshots.